src/Lykke.Job.BlockchainCashoutProcessor/Settings/MatchingEngineSettings.cs

```diff
@@ -0,0 +1,36 @@
+﻿using JetBrains.Annotations;
+using Lykke.SettingsReader.Attributes;
+using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Text;
+
+namespace Lykke.Job.BlockchainCashoutProcessor.Settings
+{
+    [UsedImplicitly]
+    public class MatchingEngineSettings
+    {
+        [UsedImplicitly(ImplicitUseKindFlags.Assign)]
+        public IpEndpointSettings IpEndpoint { get; set; }
+    }
+
+    [UsedImplicitly]
+    public class IpEndpointSettings
```

Comment:
Multiple classes in the single file - not good practice

Reply:
fixed